PKGBUILD.source

```diff
@@ -38,7 +38,7 @@ build() {
 	_archprefix="$srcdir"/install_archprefix
 	mkdir -p "$_prefix" "$_archprefix"
 	cd "$srcdir"
-	octave -q -f --eval "$(cat <<-EOF
+	octave-cli -q -f --eval "$(cat <<-EOF
```

Comment:
I'd rather not do this. Who knows how long octave-cli will stay. This is supposed to work with the octave command.